wordpress/wp-content/plugins/memberful-wp/src/acl.php

```diff
@@ -295,11 +295,6 @@ function memberful_can_user_access_post( $user, $post ) {
     return $user ? true : false;
   }
 
-  // Grant access if user has a subscription and post or one of its terms allows access with any subscription
```

Comment:
Isn't it better to return early? I.e., if a post is viewable to any subscriber, isn't it better to check it at the beginning and return as soon as possible?